author: Sara Then

output:
  html_document:
    toc: true
    toc_depth: 4
    toc_float: true
    dev: 'svg'
  md_document:
    variant: gfm


Methods: 
In Module 7, we performed analyses on 16s data using the software pacakage, QIIME 2(Bolyen et al.2019). QIIME2 and Miniconda2 installation was first completed on our local machine. The three datasets analyzed were available on QIIME2 tutorial pages. Analyses were generated by following the steps in tutorials for Moving Pictures, FMT, and Atacoma Soil Microbiome data. For all three datasets, raw sequence data was first obtained and imported. Using QIIME2, sequences were first demultiplexed. Next, quality filtering was performed. Denoising step was conducted using QIIME2 plug-in, DADA2. Once quality filtering was done, FeatureTable summaries were generated as QIIME2 artifacts. Through the tutorials, we learned how to generate a tree for phylogenetic diversity analyses and conducted Alpha and beta diversity analysis using the q2-diversity plug-in. We also learned different ways to visualize and measure the datasets through metrics, emperor plots, and alpha rarefaction plotting. QIIME2 also has the ability to generate Taxonomic analysis and provide differential abundance testing with ANCOM. The module showcased the diverse applications QIIME2 offers in performing 16s data analysis. 

Citations: 
Bolyen E, Rideout JR, Dillon MR, Bokulich NA, Abnet CC, Al-Ghalith GA, Alexander H, Alm EJ, Arumugam M, Asnicar F, Bai Y, Bisanz JE, Bittinger K, Brejnrod A, Brislawn CJ, Brown CT, Callahan BJ, Caraballo-Rodríguez AM, Chase J, Cope EK, Da Silva R, Diener C, Dorrestein PC, Douglas GM, Durall DM, Duvallet C, Edwardson CF, Ernst M, Estaki M, Fouquier J, Gauglitz JM, Gibbons SM, Gibson DL, Gonzalez A, Gorlick K, Guo J, Hillmann B, Holmes S, Holste H, Huttenhower C, Huttley GA, Janssen S, Jarmusch AK, Jiang L, Kaehler BD, Kang KB, Keefe CR, Keim P, Kelley ST, Knights D, Koester I, Kosciolek T, Kreps J, Langille MGI, Lee J, Ley R, Liu YX, Loftfield E, Lozupone C, Maher M, Marotz C, Martin BD, McDonald D, McIver LJ, Melnik AV, Metcalf JL, Morgan SC, Morton JT, Naimey AT, Navas-Molina JA, Nothias LF, Orchanian SB, Pearson T, Peoples SL, Petras D, Preuss ML, Pruesse E, Rasmussen LB, Rivers A, Robeson MS, Rosenthal P, Segata N, Shaffer M, Shiffer A, Sinha R, Song SJ, Spear JR, Swafford AD, Thompson LR, Torres PJ, Trinh P, Tripathi A, Turnbaugh PJ, Ul-Hasan S, van der Hooft JJJ, Vargas F, Vázquez-Baeza Y, Vogtmann E, von Hippel M, Walters W, Wan Y, Wang M, Warren J, Weber KC, Williamson CHD, Willis AD, Xu ZZ, Zaneveld JR, Zhang Y, Zhu Q, Knight R, and Caporaso JG. 2019. Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2. Nature Biotechnology 37: 852–857. https://doi.org/10.1038/s41587-019-0209-9


